src/Microsoft.DotNet.Interactive/Commands/RequestValue.cs

```diff
@@ -32,7 +33,7 @@ public override Task InvokeAsync(KernelInvocationContext context)
                     if (value is { })
                     {
                         var valueType = value.GetType();
-                        var mimeType = MimeType ?? Formatter.GetPreferredMimeTypeFor(valueType);
+                        var mimeType = MimeType ?? Formatter.GetPreferredMimeTypesFor(valueType).FirstOrDefault();
```

Comment:
Again, `FirstOrDefault` makes me nervous. If we need to converge to a single one when multiple are registered, what should be the behavior? Fall back to the overall preferred MIME type if available?

I also think some MIME types are more suitable for data and some for display. In the case of `RequestValue` we might prefer a data MIME type as a convention.

Reply:
makes sense